### PR TITLE
Model tweaks for entity resolution

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/biz.py [hunk 1]
```diff
@@ -102,9 +102,27 @@ def getModelDefs(self):
                     ('buyer', ('ps:contact', {}), {
                         'doc': 'The primary contact information for the buyer.',
                     }),
+                    ('buyer:org', ('ou:org', {}), {
+                        'doc': 'The buyer org.',
+                    }),
+                    ('buyer:orgname', ('ou:name', {}), {
```
Comment (rakuy0) [inline synapse/models/biz.py:108]: I probably just missed the modeling conversation, but this seems bit redundant with `ou:org:name` and `ou:org:url` (same for the corresponding seller/madeby ones)?

File: docs/synapse/userguides/storm_ref_model_introspect.ipynb [hunk 2]
```diff
@@ -251,7 +251,7 @@
     "q = 'syn:prop:type = inet:fqdn'\n",
     "print(q)\n",
     "# Execute the query and test\n",
-    "podes = await core.eval(q, num=48, cmdr=False)"
```
Comment (rakuy0) [inline docs/synapse/userguides/storm_ref_model_introspect.ipynb:254]: Is what this asserts no longer valid?

Comment (invisig0th) [reply docs/synapse/userguides/storm_ref_model_introspect.ipynb:254]: It's a number than changes (and breaks) every time any form gains a secondary property that is an `inet:fqdn` and i'm tired of it breaking :)

File: synapse/models/biz.py (hunk 1 above)
Comment (invisig0th) [reply synapse/models/biz.py:108]: It's for entity resolution where you may have observables that contain an org name or fqdn but you dont have a corresponding org yet. recording them so they can be glued back up later.

Comment (rakuy0) [reply synapse/models/biz.py:108]: :+1:

File: docs/synapse/userguides/storm_ref_model_introspect.ipynb (hunk 2 above)
Comment (rakuy0) [reply docs/synapse/userguides/storm_ref_model_introspect.ipynb:254]: :+1:

File: synapse/tests/test_model_biz.py [hunk 3]
```diff
@@ -85,6 +91,14 @@ async def test_model_biz(self):
             self.nn(nodes[0].get('seller'))
             self.nn(nodes[0].get('purchase'))
 
+            self.nn(nodes[0].get('buyer:org'))
+            self.nn(nodes[0].get('seller:org'))
+
+            self.nn('hehehaha', nodes[0].get('buyer:orgname'))
+            self.nn('hehehaha.com', nodes[0].get('buyer:orgfqdn'))
+            self.nn('lololol', nodes[0].get('seller:orgname'))
+            self.nn('lololol.com', nodes[0].get('seller:orgfqdn'))
```
Comment (Cisphyx) [inline synapse/tests/test_model_biz.py:100]: These should be `self.eq` instead of `self.nn`

**Pull request merged**